assets/js/app.js

```diff
@@ -89,3 +87,14 @@ window.addEventListener("lb:focus", (event) => {
 window.addEventListener("lb:set_value", (event) => {
   event.target.value = event.detail.value;
 });
+
+// Other global handlers
+
+window.addEventListener("contextmenu", (event) => {
```

Comment:
Should we use lb:contextmenu for consistency?

Reply:
This is a browser event we are listening on, not our own :D

Reply:
It makes the prefix for the previous one even better!